.github/workflows/dpu-server.yaml

```diff
@@ -0,0 +1,48 @@
+name: DPU server CI
+on:
+  workflow_dispatch:
+    branches: ["v0.4.x"]
+  push:
+    branches: ["v0.4.x"]
+  pull_request: 
+    branches: ["v0.4.x"]
+jobs:
+  deployment:
+    runs-on: [self-hosted, linux, x64]
+    steps:
+    - uses: actions/checkout@v3
+    - name: Initialisation runners
+      run: /start init
+    - name: Deployment infrastructure
+      run: /start deploy
+  build_x86:
+    needs: [deployment]
+    runs-on: [self-hosted, linux, x64]  
+    steps:
+    - name: Building on the HOST
+      run: /start build_x86
+  build_dpu:
+    needs: [deployment]
+    runs-on: [self-hosted, linux, x64]  
```

Comment:
Do we build dpu part on x64?

Reply:
We have only x84 runners, but it uses dpu as well